rootfs/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM alpine:3.1
+FROM alpine:3.3
+
+# install git and configure gituser
+ENV GITHOME /home/git
+ENV GITUSER git
+# this is so the minio client (https://github.com/minio/mc) works properly
+ENV DOCKERIMAGE=1
+ENV DEIS_RELEASE 2.0.0-dev
```

Comment:
> This envvar will be modified on every release. We should probably punt this to the bottom of the Dockerfile.

That's correct but I would prefer a rebuild of the image if a variable is changed and not use the cache of the previous layers